### PR TITLE
Added watchosX64 target

### LINKED ISSUE #617: Support watchosX64 target
Also looks like we have duplicated `watchosArm32Abc` source sets.

cc: @CherryPerry

### PR DESCRIPTION
**Pull request by arkivanov**
Closes #617

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.reaktive_version = '1.1.22'
+    ext.reaktive_version = '1.1.23-t1'
```
Comment (CherryPerry) [inline build.gradle:2]: Why `-t1`?

File: includedBuild/dependencies/src/main/kotlin/com/badoo/reaktive/dependencies/Deps.kt [hunk 2]
```diff
@@ -3,7 +3,7 @@ package com.badoo.reaktive.dependencies
 object Deps {
 
     private const val kotlinVersion = "1.5.0"
```
Comment (CherryPerry) [inline includedBuild/dependencies/src/main/kotlin/com/badoo/reaktive/dependencies/Deps.kt:5]: Also we can upgrade to 1.5.10.

File: build.gradle (hunk 1 above)
Comment (arkivanov) [reply build.gradle]: Accidentally, was checking the publication, thanks for spotting!

File: includedBuild/dependencies/src/main/kotlin/com/badoo/reaktive/dependencies/Deps.kt (hunk 2 above)
Comment (arkivanov) [reply includedBuild/dependencies/src/main/kotlin/com/badoo/reaktive/dependencies/Deps.kt]: Done

**Review by CherryPerry — APPROVED** [review]

**Pull request merged**